wp-content/themes/core/pcss/base/typography/_tables.pcss

```diff
@@ -25,12 +27,16 @@
 
 .th,
 .t-content th {
-	padding: 14px 1em 12px;
+	color: var(--color-mine-shaft);
+	padding: 18px 0 18px 24px;
```

Comment:
I generally find for a vanilla, responsive as possible without a real mobile solution table that it's most robust by having equal horizontal padding with a relative unit fwiw (same for `td` below).